examples/planning/closed_loop_test.ipynb

```diff
@@ -0,0 +1,347 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Closed-Loop Evaluation\n",
+    "In this notebook you are going to evaluate a CNN-based policy to control the SDV with a protocol named *closed-loop* evaluation.\n",
+    "\n",
+    "**Note: this notebook assumes you've already run the [training notebook](./train.ipynb) and stored your model successfully.**\n",
+    "\n",
+    "## What is a closed-loop evaluation?\n",
+    "In closed-loop evaluation the model is in **full control of the SDV**. At each time step, we predict the future trajectory and then move the AV to the first of the model's predictions. \n",
+    "\n",
+    "We refer to this process with the terms **forward-simulate** or **unroll**.\n",
+    "\n",
+    "![closed-loop](../../images/planning/closed-loop.svg)\n",
+    "\n",
+    "\n",
+    "## What can we use closed-loop evaluation for?\n",
+    "Closed-loop is crucial to assess the real performance of a model. **Ideally, we would test the model on the road in the real world**. However, this is clearly very expensive and scales poorly. Forward-simulation is an attempt to evaluate the system in a setting which is as close as possible to a real road test on the same route.\n",
```

Comment:
'real performance' sounds a bit weird -> to assess a model's capabilities of deploying it in the real world or something like that.